flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java

```diff
@@ -1409,7 +1409,7 @@ public void testDropDueToLatenessSliding() throws Exception {
 	}
 
 	@Test
-	public void testDropDueToLatenessSessionZeroLateness() throws Exception {
+	public void testDropDueToLatenessSessionZeroLatenessPurgingTrigger() throws Exception {
 		final int GAP_SIZE = 3;
 		final long LATENESS = 0;
 
```

Comment:
Here the name changes to indicate that it is purging, but the Trigger is the same (EventTime) without purging.